### PR TITLE
Rename boundary to face

### PR DESCRIPTION
**Pull request by KristofferC**

### CODE REVIEW

**Comment by KristofferC** [comment]
What's up with 0.5?

File: src/grid.jl [hunk 1]
```diff
@@ -33,17 +33,37 @@ end
 (::Type{Cell{dim}}){dim,N}(nodes::NTuple{N}) = Cell{dim,N}(nodes)
 
 # Typealias for commonly used cells
+
+to_interpolation(c::Cell) = to_interpolation(typeof(c))
+
+getfacelist(c::Cell)     = getfacelist(typeof(c))
+getfacelist(c::Type{<:Cell}) = getfacelist(to_interpolation(c))
```
Comment (fredrikekre) [inline src/grid.jl]: You can't use this syntax on 0.5. That's the test failure.

Comment (KristofferC) [inline src/grid.jl]: oh...

Comment (KristofferC) [inline src/grid.jl]: This wasnt even supposed to be in here

File: docs/src/lib/utility_functions.md [hunk 2]
```diff
@@ -49,14 +49,14 @@ function_divergence
 spatial_coordinate
 ```
 
-## BoundaryValues
+## FaceValues
 
-All of the methods for [`CellValues`](@ref) apply for `BoundaryValues` as well.
-In addition, there are some methods that are unique for `BoundaryValues`:
+All of the methods for [`CellValues`](@ref) apply for `FaceValues` as well.
+In addition, there are some methods that are unique for `FaecValues`:
```
Comment (fredrikekre) [inline docs/src/lib/utility_functions.md]: `FaecValues` -> `FaceValues`

**Comment by fredrikekre** [comment]
I am surprised this rename is such a big diff :p

**Comment by KristofferC** [comment]
Don't bother review this. I have to play around a bit to see what makes sense.

**Comment by fredrikekre** [comment]
Sure, feel free to request a review when it is more finished.

**Comment by fredrikekre** [comment]
#125 ?

**Comment by KristofferC** [comment]
Yes

**Pull request closed without merging**